### PR TITLE
Build campaigns/applications pages in profile

### PR DESCRIPTION
**Pull request by webdeveloper001**

### CODE REVIEW

File: app/shared/buttons/BlackOutlinedButtonClient.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Button from '@mui/material/Button';
```
Comment (tomer-tgp) [inline app/shared/buttons/BlackOutlinedButtonClient.js:2]: no mui button

File: app/(user)/profile/campaigns/components/ApplicationSection.js [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+        console.log(error)
+    }
+}
+
+function ApplicationSection() {
+    const router = useRouter();
+    const [applications, setApplications] = useState([]);
+    const loadApplications = async () => {
+        const res = await loadApplicationsCallback();
+        if(applications.length === 0) {
```
[the remark this answers is not in the document]
Comment (tomer-tgp) [reply app/(user)/profile/campaigns/components/ApplicationSection.js:72]: Is this correct @webdeveloper001

File: app/(user)/profile/campaigns/components/CampaignStaff.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+import React from 'react';
+import gpApi from 'gpApi';
+import gpFetch from 'gpApi/gpFetch';
+import PortalPanel from '@shared/candidate-portal/PortalPanel';
+import StaffCard from './StaffCard';
+
+async function loadStaff() {
+    try {
+        const api = gpApi.campaign.staff.userStaff;
+        console.log(api);
```
Comment (tomer-tgp) [inline app/(user)/profile/campaigns/components/CampaignStaff.js:16]: remove logs

File: app/shared/buttons/BlackOutlinedButton.module.scss [hunk 4]
```diff
@@ -0,0 +1,20 @@
+.button {
```
Comment (tomer-tgp) [inline app/shared/buttons/BlackOutlinedButton.module.scss:1]: do you need this file now that we are not using mui button?

File: app/shared/buttons/BlackOutlinedButtonClient.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import styles from './BlackOutlinedButton.module.scss';
```
Comment (tomer-tgp) [inline app/shared/buttons/BlackOutlinedButtonClient.js:2]: no need

**Review by tomer-tgp — CHANGES_REQUESTED** [review]

File: app/shared/buttons/BlackOutlinedButton.module.scss (hunk 4 above)
Comment (webdeveloper001) [reply app/shared/buttons/BlackOutlinedButton.module.scss:1]: yeah we need that

File: app/shared/buttons/BlackOutlinedButtonClient.js (hunk 5 above)
Comment (webdeveloper001) [reply app/shared/buttons/BlackOutlinedButtonClient.js:2]: I think we need that

**Review by tomer-tgp — APPROVED** [review]

**Pull request merged**